python/llm/src/bigdl/llm/transformers/convert.py

```diff
@@ -932,4 +932,16 @@ def safe_bmm_fwd(*args, **kwargs):
         convert_forward(model,
                         module.RwkvSelfAttention,
                         rwkv_attention_forward)
+    elif model.config.model_type == "deci":
+        modeling_module_name = model.__class__.__module__
+        module = importlib.import_module(modeling_module_name)
+        convert_forward(model,
+                        module.LlamaRMSNorm,
+                        llama_rms_norm_forward)
+        convert_forward(model,
+                        module.LlamaMLP,
+                        llama_mlp_forward)
+        convert_forward(model,
```

Comment:
Logits are different if using this, we can't use llama's attention forward directly.

Reply:
fixed with decilm_attention_forward_4_35_2